internal/daemon/daemon.go

```diff
@@ -213,12 +350,17 @@ func (d *Daemon) prepareCni() error {
 	return nil
 }
 
-func (d *Daemon) runSideManager(mgr SideManager, managerCtx context.Context) error {
+func (d *Daemon) runSideManager(mgr SideManager, identifier string, managerCtx context.Context) error {
 	err := mgr.StartVsp(managerCtx)
 	if err != nil {
 		d.log.Error(err, "Failed to start VSP in sideManager")
 		return err
 	}
+
+	// Mark the DPU as ready after successful VSP start
+	// TODO: hook this up to a health probe to the VSP so that it goes into NotReady while it doesn't respond
+	d.updateDpuStatus(identifier, "Ready")
```

Comment:
During the demo, you mentioned it would always be NotReady right now. Does this mean that this doesn't work? It looks fine to me.

Reply:
ok makes sense.